### DISCUSSION TITLE
Integration of Third-Party Authentication Providers

### DISCUSSION THREAD

**Discussion by chums122** [pelican-dev/panel#92] [Ideas]
### Feature Suggestion:

To improve security and enhance user convenience, it would be beneficial to integrate additional third-party authentication providers into the application. Support for popular OAuth2 providers such as ADFS, Entra ID/Azure AD, and Authentik, as well as LSAP/AD, could significantly enrich the application's functionality.

**Anticipated Benefits:**

- **Enhanced Security:** Using established external authentication providers can fortify the application’s security measures.
- **Simplified User Access:** Allowing users to log in with recognized OAuth2 accounts would streamline their experience.

Additionally, while primarily aimed at improving user experience, this enhancement could also serve as a foundation for reaching enterprise users in the future, should there be interest in expanding to that market.

**Comment by lancepioch** [comment]
Thanks, we are actively working on providing Oauth support!
